### PR TITLE
Chore: update caniuse

### PR DESCRIPTION
**Pull request by ryantxu**
the build keeps asking me to update `caniuse-lite`.... so I did

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,20 +67,20 @@
...
+    "@babel/plugin-proposal-class-properties": "7.13.0",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.13.8",
+    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
     "@babel/plugin-proposal-optional-chaining": "7.13.12",
-    "@babel/plugin-syntax-dynamic-import": "7.8.3",
-    "@babel/plugin-proposal-class-properties": "7.13.0",
     "@babel/plugin-proposal-private-methods": "7.13.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.13.8",
+    "@babel/plugin-syntax-dynamic-import": "7.8.3",
     "@babel/plugin-transform-react-constant-elements": "7.13.13",
```
Comment (torkelo) [inline package.json:76]: These looks like they are from my babel PRs? 

https://github.com/grafana/grafana/pull/32534
https://github.com/grafana/grafana/pull/32534

File: package.json [hunk 2]
```diff
@@ -67,20 +67,20 @@
...
     "@babel/preset-env": "7.13.12",
     "@babel/preset-react": "7.13.13",
     "@babel/preset-typescript": "7.13.0",
     "@grafana/api-documenter": "7.11.2",
     "@grafana/api-extractor": "7.10.1",
     "@grafana/eslint-config": "2.3.0",
+    "@rtsao/plugin-proposal-class-properties": "7.0.1-patch.1",
```
Comment (torkelo) [inline package.json:83]: dont need this one anymore removed in https://github.com/grafana/grafana/pull/32559

File: package.json (hunk 1 above)
Comment (ryantxu) [reply package.json:76]: I think they are artifacts of anything that touches package.json automatically.

I will remove them from this PR so you can update in the babel PRs

**Comment by ryantxu** [comment]
@torkelo I removed the changes to package.json -- and this is only yarn.lock updates.  That will make merging your other PRs easy

**Review by torkelo — APPROVED** [review]

**Pull request merged**